resources/models.py

```diff
@@ -197,8 +197,8 @@ class WinterStorageArea(AbstractArea, TranslatableModel):
     # People just queue for these, then as long as there s still space
     # next person in the queue can put his/her boat there.
     # The area is separated into sections, that limit the length of the suitable boat.
-    number_of_section_spaces = models.PositiveSmallIntegerField(
-        verbose_name=_("number of section places"), null=True, blank=True
+    estimated_number_of_section_spaces = models.PositiveSmallIntegerField(
```

Comment:
we also need to add these to the schema 🤔

Reply:
nvm, looked at the `WinterStorageSectionNode` 🤦  all good, ws area has those automatically